src/lexfloatclient.ts

```diff
@@ -245,6 +245,20 @@ export class LexFloatClient {
 		throw new LexFloatClientException(status);
 	}
 
+		/**
+	 * Gets the library version.
+	 * @returns Returns the library version.
+	 * @throws {LexFloatClientException}
+	 */
+		static GetFloatingClientLibraryVersion(): string {
+			const array = new Uint8Array(256);
+			const status = LexFloatClientNative.GetFloatingClientLibraryVersion(array, array.length);
+			if (status != LexFloatStatusCodes.LF_OK) {
+				throw new LexFloatClientException(status);
+			}
+			return arrayToString(array);
+		}
+
 	/**
```

Comment:
@mumin-khan Please check if the format of this function is correct. Usually, it is a UI issue but please confirm it.